arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/fragment/stack/CancunStackFragment.java

```diff
@@ -51,4 +52,9 @@ protected void traceMcopyFamily(Trace.Hub trace, InstructionFamily currentInstFa
   protected void traceTransientFamily(Trace.Hub trace, InstructionFamily currentInstFamily) {
     trace.pStackTransFlag(currentInstFamily == TRANSIENT);
   }
+
+  @Override
+  protected void traceMxpFlag(Trace.Hub trace, OpCodeData opCodeData) {
+    trace.pStackMxpFlag(opCodeData.isMxp());
```

Comment:
Why call this one `opCodeData.isMxp()` when we also have a `opCodeData.isMxpLondon()` ? IIRC opcode data is extracted from a London/Shanghai/Cancun  `opcodes.yml` file, no ? Is something preventing from using a unified interface ?